systemtest/src/test/java/io/strimzi/systemtest/security/oauth/OauthPlainIsolatedST.java

```diff
@@ -73,6 +83,8 @@ public class OauthPlainIsolatedST extends OauthAbstractST {
 
     private final String oauthClusterName = "oauth-cluster-plain-name";
     private final String customClaimListenerPort = "9099";
+    private final List<String> expectedOauthMetrics = Arrays.asList("strimzi_oauth_http_requests_maxtimems", "strimzi_oauth_http_requests_mintimems",
+        "strimzi_oauth_http_requests_avgtimems", "strimzi_oauth_http_requests_totaltimems", "strimzi_oauth_http_requests_count");
```

Comment:
Actually, whenever OAuth client authentication is configured with something other that `access_token` there should also be `strimzi_oauth_authentication_requests_maxtimems`, `strimzi_oauth_authentication_requests_mintimems`, `strimzi_oauth_authentication_requests_avgtimems`, `strimzi_oauth_authentication_requests_totaltimems`, and `strimzi_oauth_authentication_requests_count`.

Reply:
Or actually, for the client oauth these `*_authentication_requests_*` metrics should always be present, and the `*_http_requests_*` should be present when something other than `access_token` is configured.

Reply:
So I tested all of the metrics you mentioned here, all components contained `*_http_requests_*` and `*_authentication_requests_*`, just Kafka didn't contain `*_authentication_requests_*` metrics.

I think we can check just the `*_http_requests_*`, as it seems that those will be present everytime.

WDYT @mstruk ?